wpt

```diff
@@ -4,8 +4,10 @@ if __name__ == "__main__":
     from tools.wpt import wpt
     from sys import version_info, argv, exit
     args, extra = wpt.parse_args(argv[1:])
+    commands = wpt.load_commands()
+    py3only = commands[args.command]["py3only"]
 
-    if args.py3 and version_info.major < 3:
+    if (args.py3 or py3only) and version_info.major < 3:
         from subprocess import call
         exit(call(['python3', argv[0]] + [args.command] + extra))
```

Comment:
Might be good to improve the error handling here if `python3` isn't on the system, since there was no explicit opt-in.